modular_skyrat/master_files/code/modules/mob/living/examine_tgui.dm

```diff
@@ -57,16 +57,24 @@
 	var/headshot = ""
 
 	//  Handle OOC notes first
-	if(preferences && preferences.read_preference(/datum/preference/toggle/master_erp_preferences))
-		var/e_prefs = preferences.read_preference(/datum/preference/choiced/erp_status)
-		var/e_prefs_nc = preferences.read_preference(/datum/preference/choiced/erp_status_nc)
-		var/e_prefs_v = preferences.read_preference(/datum/preference/choiced/erp_status_v)
-		var/e_prefs_mechanical = preferences.read_preference(/datum/preference/choiced/erp_status_mechanics)
-		ooc_notes += "ERP: [e_prefs]\n"
-		ooc_notes += "Non-Con: [e_prefs_nc]\n"
-		ooc_notes += "Vore: [e_prefs_v]\n"
-		ooc_notes += "ERP Mechanics: [e_prefs_mechanical]\n"
-		ooc_notes += "\n"
+	if(preferences)
+		if(preferences.read_preference(/datum/preference/toggle/master_erp_preferences))
+			var/e_prefs = preferences.read_preference(/datum/preference/choiced/erp_status)
+			var/e_prefs_nc = preferences.read_preference(/datum/preference/choiced/erp_status_nc)
+			var/e_prefs_v = preferences.read_preference(/datum/preference/choiced/erp_status_v)
+			var/e_prefs_mechanical = preferences.read_preference(/datum/preference/choiced/erp_status_mechanics)
+			ooc_notes += "ERP: [e_prefs]\n"
+			ooc_notes += "Non-Con: [e_prefs_nc]\n"
+			ooc_notes += "Vore: [e_prefs_v]\n"
+			ooc_notes += "ERP Mechanics: [e_prefs_mechanical]\n"
+			ooc_notes += "\n"
+
+		if(!CONFIG_GET(flag/disable_antag_opt_in_preferences))
+			var/antag_prefs = holder.mind?.ideal_opt_in_level
+			if(isnull(antag_prefs))
+				antag_prefs = preferences.read_preference(/datum/preference/choiced/antag_opt_in_status)
```

Comment:
uses the ideal pref because this is, to my knowedlgfe, used more as an advisory setting, a way to gauge what the PLAYER wants